src/TextField/EnhancedTextarea.js

```diff
@@ -46,46 +45,42 @@ const EnhancedTextarea = React.createClass({
     textareaStyle: React.PropTypes.object,
     value: React.PropTypes.string,
     valueLink: React.PropTypes.object,
-  },
+  };
 
-  contextTypes: {
-    muiTheme: React.PropTypes.object.isRequired,
-  },
+  static defaultProps = {
+    rows: 1,
+  };
 
-  getDefaultProps() {
-    return {
-      rows: 1,
-    };
-  },
+  static contextTypes = {
+    muiTheme: React.PropTypes.object.isRequired,
+  };
 
-  getInitialState() {
-    return {
-      height: this.props.rows * rowsHeight,
-    };
-  },
+  state = {
+    height: this.props.rows * rowsHeight,
```

Comment:
As discussed -- will refine later, can't make it all perfect in one go. Plus, I think that in some cases, if it really is the "initial" state that component should have, and that it is a prop that is unlikely to change and be dynamic, that accessing it in the state initializer could be a valid use case.